cernopendata/modules/fixtures/data/docs/cms-guide-troubleshooting/cms-guide-troubleshooting.md

```diff
@@ -1,34 +1,77 @@
-- The root password for CMS Open Data VM is `password`
+This page lists possible solutions to frequently encountered issues with CMS Open Data, for the virtual machine, for the CMSSW software and examples and for the file download and access.
```

Comment:
This page lists possible solutions to frequently encountered issues with CMS Open Data: for the virtual machine; for the CMSSW software and examples; and for the file download and access.

?